src/aml/bert.py

```diff
@@ -0,0 +1,203 @@
+
+            label_list.append(aos_list_per_review)
+
+    return reviews_list, label_list
+
+
+def save_train_reviews_to_file(original_reviews: List[Review], output: str) -> None:
+    train, _ = convert_reviews_from_lady(original_reviews)
+    dev, _ = convert_reviews_from_lady(original_reviews)
+
+    if not os.path.isdir(output): 
+        os.makedirs(output)
+
+    path_witout_end_dot = output[:-1]
+    os.mkdir(path_witout_end_dot)
+
+    for h in range(0, 101, 10):
+        output = f'{path_witout_end_dot}/latency-{h}'
+
+        os.mkdir(output)
+
+        with open(f'{output}/dev.txt', 'w', encoding='utf-8') as file:
+            for d in dev:
+                file.write(d + '\n')
+
+        with open(f'{output}/train.txt', 'w', encoding='utf-8') as file:
+            for d in train:
+                file.write(d + '\n')
+
+def save_test_reviews_to_file(validation_reviews: List[Review], h_ratio: int, output: str) -> None:
+    path_witout_end_dot = output[:-1]
+
+    if not os.path.isdir(output):
+        raise Exception(f'Output path {output} does not exist')
+    
+    _, labels = convert_reviews_from_lady(validation_reviews)
+
+
+    for h in range(0, h_ratio * 100 + 1, 10):
+        hp = h / 100
+
+        path = f'{path_witout_end_dot}/latency-{h}'
+
+        if not os.path.isdir(path):
+            os.makedirs(path)
+
+        test_hidden = []
+        for t in range(len(validation_reviews)):
+            if random.random() < hp:
+                test_hidden.append(validation_reviews[t].hide_aspects(mask='z', mask_size=5))
+            else:
+                test_hidden.append(validation_reviews[t])
+        preprocessed_test, _ = convert_reviews_from_lady(test_hidden)
+
+        with open(f'{path}/test.txt', 'w', encoding='utf-8') as file:
+            for d in preprocessed_test:
+                file.write(d + '\n')
+
+        pd.to_pickle(labels, f'{path}/test-labels.pk')
+#--------------------------------------------------------------------------------------------------
+# Class Definition
+#--------------------------------------------------------------------------------------------------
+
+# TODO: Change these
+# @inproceedings{DBLP:conf/www/YanGLC13,
+#   author       = {Xiaohui Yan and Jiafeng Guo and Yanyan Lan and Xueqi Cheng},
+#   title        = {A biterm topic model for short texts},
+#   booktitle    = {22nd International World Wide Web Conference, {WWW} '13, Rio de Janeiro, Brazil, May 13-17, 2013},
+#   pages        = {1445--1456},
+#   publisher    = {International World Wide Web Conferences Steering Committee / {ACM}},
+#   year         = {2013},
+#   url          = {https://doi.org/10.1145/2488388.2488514},
+#   biburl       = {https://dblp.org/rec/conf/www/YanGLC13.bib},
+# }
+class BERT(AbstractAspectModel):
+    def __init__(self, naspects, nwords): 
+        super().__init__(naspects, nwords)
+
+    # TODO: What is this? should I change it?
+    def load(self, path):
```

Comment:
@3ripleM this is for test purpose as we only need to load the model if it's already there. if you don't need and use the baseline code for this, as I see blow, simply remove it.